opentelemetry-sdk/tests/metrics/test_aggregation.py

```diff
@@ -83,12 +83,10 @@ def test_aggregate(self):
         explicit_bucket_histogram_aggregation.aggregate(8)
         explicit_bucket_histogram_aggregation.aggregate(9999)
 
-        self.assertEqual(explicit_bucket_histogram_aggregation.value[0], -1)
-        self.assertEqual(explicit_bucket_histogram_aggregation.value[5], 2)
-        self.assertEqual(explicit_bucket_histogram_aggregation.value[10], 15)
-        self.assertEqual(
-            explicit_bucket_histogram_aggregation.value[inf], 9999
-        )
+        self.assertEqual(explicit_bucket_histogram_aggregation.value[0], 1)
+        self.assertEqual(explicit_bucket_histogram_aggregation.value[5], 1)
+        self.assertEqual(explicit_bucket_histogram_aggregation.value[10], 2)
+        self.assertEqual(explicit_bucket_histogram_aggregation.value[inf], 1)
```

Comment:
Are the resulting values different because of the change in how histograms are searched? Or was it just broken before?

Reply:
If the latter, why wasnt this failing the CI?

Reply:
It was broken before :sweat_smile: